src/auth-restrict.md

```diff
@@ -0,0 +1,302 @@
+Enforcing Advanced Login and Connection Restrictions
+====================================================
+
+A feature of Guacamole as of version 1.6.0 is an extension that allows you to
+enforce advanced restrictions on both user logins to Guacamole as well as the
+user of connections and connection groups. The extension,
+`guacamole-auth-restrict`, decorates other authentication extensions that
+contain user information and/or connections, and allows you to apply
+restrictions to those objects for the time(s) that users are allowed or not
+allowed to log in, the hosts from which users may or may not log in, the
+time(s) that certain connections and balancing connection groups may or
+may not be used, and the hosts from which certain connections and
+balancing connection groups may or not may not be used. The goal is to
+give administrators of a Guacamole system additional flexibility in
+restricting when and from where the system can be used.
+
+As this extension decorates underlying extensions, it must be used alongside
+one that is capable of storing additional information for users, user groups,
+connections, and connection groups. Currently the only extension provided
+by the Guacamole project that is capable of doing this is the
+[JDBC authentication extension](jdbc-auth). Other extensions, such as the
+[LDAP authentication extension](ldap-auth) or
```

Comment:
Are you saying that the LDAP and JSON extensions _are_ currently capable of being used with this extension (or might some day be?) If it's that they _might_ someday work, I'd just leave that bit out. I don't see a lot of benefit to speculatation on what features might theoretically be added to these in the future. If they would currently work, I'd be more clear about that.

Reply:
Fair point. I've removed it.